src/ontology/mondo-edit.obo

```diff
@@ -171877,7 +171877,7 @@ is_a: MONDO:0003847 {source="OMIMPS:158000"} ! hereditary disease
 is_a: MONDO:0005093 {source="https://orcid.org/0000-0001-5208-3432"} ! skin disorder
 relationship: has_characteristic HP:0000006 {source="MONDO:HPOA", source="OMIM:158000", source="Orphanet:573"} ! Autosomal dominant inheritance
 relationship: has_characteristic MONDO:0021152 {source="OMIMPS:158000"} ! inherited
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6463 {source="OMIM:158000"} ! KRT86
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6463 ! KRT86
```

Comment:
this source looks like it should be maintained https://www.omim.org/entry/158000

Reply:
done